grub2/get-option

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/perl
+
+# usage: get-option OPTION
+#
+# Read boot option.
+#
+# OPTION is either of the form 'key=value' or 'key="value"' or just 'key'.
+#
+# The option value is written into the file passed via 'PBL_RESULT' environment var.
+# Either 'option=value', or 'option', or nothing is returned.
+
+use strict;
+
+my $file = "/etc/default/grub";
+
+my $opt = shift;
+
+my $opt_name = $opt;
+
+if($opt_name =~ s/=("?).*//) {
```

Comment:
No whitespace in the regexp? Not maybe `s/=\s*("?).*//` ?

Reply:
Not allowed.